base_layer/core/src/transactions/transaction_components/transaction.rs

```diff
@@ -97,19 +92,19 @@ impl Transaction {
 
     /// Returns the height of the minimum height where the transaction is spendable. This is calculated from the
     /// transaction kernel lock_heights and the maturity of the input UTXOs.
-    pub fn min_spendable_height(&self) -> u64 {
+    pub fn min_spendable_height(&self) -> Result<u64, TransactionError> {
         self.body.min_spendable_height()
     }
 
-    /// This function adds two transactions together. It does not do cut-through. Calling Tx1 + Tx2 will result in
-    /// vut-through being applied.
+    /// This function adds two transactions together. It does not do cut-through but simply sums up the offsets and
+    /// extends all inputs, outputs and kernels.
     pub fn add_no_cut_through(mut self, other: Self) -> Self {
```

Comment:
```suggestion
    pub fn add(mut self, other: Self) -> Self {
```
I dont see why we need to reference cut-through here as Tari does not use or support the use of cut-through anymore

Reply:
perhaps best to just delete this and use the impl add